README.md

```diff
@@ -1,58 +1,144 @@
 ![Banner](Documentation/Images/Banner.png)
 ## Co-op multiplayer RPG and utilities built with Unity Netcode for GameObjects
 
-|[Feedback Form](#feedback-form) |
-| -- |
+Boss Room is a fully functional co-op multiplayer RPG made with Unity Netcode. It is an educational sample designed to showcase typical gameplay [patterns](https://docs-multiplayer.unity3d.com/netcode/current/learn/bossroom/bossroom-actions/index.html) that are frequently featured in similar networked games.
+
+
+__PLEASE NOTE__:
+Boss Room is compatible with the latest Unity Long Term Support (LTS) editor version, currently [2021 LTS](https://unity.com/releases/2021-lts). Please include standalone support for Windows/Mac in your installation. <br>
+
+**Boss Room has been developed and tested on the following platforms:**
+- Windows
+- Mac
+- iOS
+- Android <br>
 
-| Support is available on [Discord](https://discord.gg/mNgM2XRDpb) and [forums](https://forum.unity.com/forums/multiplayer.26/) to help you work through issues you may encounter when using Boss Room. |
+**Boss Room has been tested for use on the following devices:**
+- iPhone 6S
+- Pixel 3 <br>
```

Comment:
@jilfranco-unity what's the device you're testing with again? That's the J3 right?

Reply:
The J2!